source/documentation/annexes.md

```diff
@@ -37,6 +37,12 @@ You can work on a dataset that is bigger than your memory by reading in a bit of
 
 Our laptops tend to have only 8GB or 16GB, so it's an advantage of the AP that we can offer more. We are open to increasing the maximum memory, so let us know if you need more, to help us justify the additional cost.
 
+### RStudio memory
+
+To understand what is taking up the memory, you can try [pryr](https://rdrr.io/cran/pryr/man/mem_used.html).
+
+To free up memory in RStudio indiscriminately, do a [hard reset of your RStudio environment](tools.html#hard-reset-of-your-environment)
```

Comment:
Is there anything users should be aware of when doing this. Could there be any adverse effects?

```suggestion
To free up memory in RStudio indiscriminately, do a [hard reset of your RStudio environment](tools.html#hard-reset-of-your-environment).
```